docker-compose.yaml

```diff
@@ -29,4 +30,22 @@ services:
     depends_on:
       - db
       - redis
-      
+      # - datadog
+  # agent section
+  datadog:
+    image: datadog/docker-dd-agent
+    restart: always
+    ports:
+      - "8126:8126"
+      - "8125:8125/udp"
+    links:
```

Comment:
links are legacy feature and it is deprecated. We already have a network between containers.

Reply:
no issues, the deployment is in heroku, so this wont affect

Reply:
Why not remove it then?

Reply:
I haven't tested without links, shall do it later. A bit backwards in awareness with docker features.